cypress/integration/searchPageView_spec.js

```diff
@@ -18,10 +18,9 @@ describe("User visits the search page view", () => {
 
   it("should be able to search for a movie given an input", () => {
     cy.get("input")
-      .type("Bill")
+      .type("Rogue")
+      .get('.search-button').click()
       .get(".movie-container")
-      .children()
```

Comment:
It looks like children() was causing some issues. Good move to reach for a more exact command with the .get().